src/utils.js

```diff
@@ -75,9 +100,104 @@ const KNK_MOVE_LIST = ['Opening Ceremony', 'Kesa-gatame', 'Kata-gatame', 'Kami-s
 const JNK_MOVE_LIST = ['Opening Ceremony', 'Tsuki-dashi', 'Kata-oshi', 'Ryote-dori', 'Kata-mawashi', 'Ago-oshi', 'Kiri-oroshi', 'Ryokata-oshi', 'Nanami-uchi', 'Katate-dori', 'Katate-age', 'Obi-tori', 'Mune-oshi', 'Tsuki-age', 'Uchi-oroshi', 'Ryogan-tsuki', 'Closing Ceremony'];
```

Comment:
NOTE:  I ultimately think all these move lists should come from a JSON file, perhaps one per kata.

Reply:
👍🏼

Reply:
I can do this as I will know instantly if it worked or not...